docs/runbooks/HPPNotReady.md

```diff
@@ -2,31 +2,38 @@
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-The HPPNotReady alert is indicative of a HPP installation being in a degraded state;  
+The `hostpath-provisioner` (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
+The HPPNotReady alert indicates that a HPP installation is in a degraded state:
```

Comment:
"a HPP" > "an HPP"

Reply:
My bad, I somehow took to reading H as "heitch" >_<